platform/api/aws/ec2.go

```diff
@@ -105,7 +107,19 @@ func (a *API) CreateInstances(name, keyname, userdata string, count uint64) ([]*
 		},
 	}
 
-	reservations, err := a.ec2.RunInstances(&inst)
+	var reservations *ec2.Reservation
+	err = util.RetryConditional(5, 5*time.Second, func(err error) bool {
```

Comment:
Is this going to end up calling the other util.WaitUntilReady in a loop? If so that'll explode the time this could wait for.

Reply:
No, the check function is just a validation of the error & the run function is a direct call to AWS.